MOM_input

```diff
@@ -388,9 +355,6 @@ ETA_TOLERANCE = 1.0E-06         !   [m] default = 3.75E-09
                                 ! total tolerance for SSH is 4 times this value.  The default is
                                 ! 0.5*NK*ANGSTROM, and this should not be set less than about
                                 ! 10^-15*MAXIMUM_DEPTH.
-VELOCITY_TOLERANCE = 1.0E-04    !   [m s-1] default = 3.0E+08
```

Comment:
Interesting that GFDL are only checking `ETA_TOLERANCE` (via default `ETA_TOLERANCE` = 1.0E-06), not `VELOCITY_TOLERANCE`. This seems potentially risky -  is there any harm in retaining `VELOCITY_TOLERANCE = 1.0E-04 `?